sources/europe/fr/PCRS.geojson

```diff
@@ -0,0 +1,2288 @@
+{
+    "type": "Feature",
+    "properties": {
+        "id": "fr.pcrs",
+        "name": "PCRS raster",
+        "type": "tms",
+        "url": "https://wxs.ign.fr/ortho/geoportail/wmts?SERVICE=WMTS&REQUEST=GetTile&VERSION=1.0.0&LAYER=PCRS.LAMB93&STYLE=normal&FORMAT=image/jpeg&tileMatrixSet=LAMB93_5cm_EPSG&tileMatrix={zoom}&tileRow={y}&tileCol={x}",
```

Comment:
While normally it is preferred to use a WMTS server, the projection of the TileMatrix is (as far as I know) not supported by all OSM editors. The WMS server mentioned here https://geoservices.ign.fr/services-web-experts-ortho seems to support EPSG:4326, which is widely supported: 
```
"type": "wms",
"url": "https://wxs.ign.fr/ortho/geoportail/r/wms?LAYERS=PCRS+LAMB93&STYLES=&CRS={proj}&BBOX={bbox}&FORMAT=image/jpeg&WIDTH={width}&HEIGHT={height}&VERSION=1.3.0&SERVICE=WMS&REQUEST=GetMap",
"available_projections": ["EPSG:3857","EPSG:4326"]
```

Reply:
Sounds good thank you